languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml

```diff
@@ -38542,6 +38542,42 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
 
 
     <category id='COMPOUNDING' name="Palavras Compostas" type="misspelling">
+        <rulegroup id="X_RAY_MACHINE" name="Raio X com hífen" default="temp_off">
+            <rule>
+                <pattern>
+                    <token case_sensitive="yes" regexp="yes">[rR]aios?</token>
+                    <token case_sensitive="no">X</token>
```

Comment:
`case_sensitive="no"` is not necessary

Reply:
I know, but I wanted to make it explicit, since in other cases it is.